docs/guide/setup-examples/remote_dev.md

```diff
@@ -114,19 +131,19 @@ function rsync_git() {
     # Async is important to not block git checkout
     echo "Async rsync of git repo $repo to $target/$repo ..."
 
-    rsync --verbose -rd --del --exclude='.git/' --filter=':- .gitignore' \
+    rsync --verbose -rd --exclude='.git/' --filter=':- .gitignore' \
         . "$target/$repo" &
 }
 
 export -f rsync_git
 ```
 
-This allows me to run `rsync_git`
-when I'm at the root of my repo
-to upload the whole folder to my server.
-Since I want to have the code on my server
-to be always a mirror of the code on my laptop,
-I added git hooks to automatically run `rsync_git` when I checkout a branch:
+With this you can run `rsync_git`
+when you're at the root of the repo
+to upload the whole folder to your server.
+
+To keep the remote code always a mirror of the code on your laptop,
+you want to automatically run `rsync_git` when you checkout a branch:
 
 ```sh
 echo "source ~/.zshrc; rsync_git" > .git/post-checkout
```

Comment:
It should be noted that hooks executed according to their shebang line and I'm not sure if they use `$SHELL` as the default if none exists. Since you're sourcing your zshrc, you'd probably want it to be interpreted by zsh.